server/src/main/resources/configuration.yml

```diff
@@ -3,8 +3,10 @@ vertxOptions:
   workerPoolSize: 1
   internalBlockingPoolSize: 1
 
-deploymentOptions:
-  instances: 1
+restVerticleDeploymentOptions:
+  instances: 2
```

Comment:
in case of worker pool based, what does having multiple instances mean?

Reply:
in worker pool, instances are no concurrent in terms of execution.